pkg/controllers/vdb/install_reconciler.go

```diff
@@ -69,11 +70,33 @@ func (d *InstallReconciler) Reconcile(ctx context.Context, req *ctrl.Request) (c
 	if err := d.PFacts.Collect(ctx, d.Vdb); err != nil {
 		return ctrl.Result{}, err
 	}
-	return d.analyzeFacts(ctx)
+	// We generate https conf file and skip the install phase when running
+	// the vclusterOps feature flag
+	if vmeta.UseVClusterOps(d.Vdb.Annotations) {
+		return d.installForVClusterOps(ctx)
+	}
+
+	return d.installForAdmintools(ctx)
+}
+
+// installForVClusterOps will go through the install phase for vclusterOps.
+// It only generates the http certs and requeue if at least one pod has not done the install
+func (d *InstallReconciler) installForVClusterOps(ctx context.Context) (ctrl.Result, error) {
+	hasUninstalledPods, err := d.generateHTTPCerts(ctx)
+	if err != nil {
+		return ctrl.Result{}, err
+	}
+	if hasUninstalledPods {
+		// We do not proceed to the next actor until
+		// all pods have done the install
```

Comment:
Lets log a message. I like to have a clear reason each time we requeue a reconcile iteration.